gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -373,6 +382,18 @@ public GoogleCloudStorageImpl(
             ? null
             : this.storage.getRequestFactory().getInitializer();
 
+    if (this.storageOptions.isMetricsEnabled()) {
+      checkArgument(
+          httpRequestInitializer instanceof RetryHttpInitializer,
+          "request initializer must be an instance of the RetryHttpInitializer class"
+              + " when gRPC API enabled");
+      Credentials credentials = ((RetryHttpInitializer) httpRequestInitializer).getCredentials();
```

Comment:
This could be inlined when ternary operator is used

Reply:
It's a multi-line conditional code, ternary operator cannot be used.